src/modules/layout/components/Snackbar.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+
+import React from 'react'
+import { Animated } from 'react-native'
+import styled, { type StyledComponent } from 'styled-components/native'
+import type { ThemeType } from '../../theme/constants/theme'
+
+export const SNACKBAR_HEIGHT = 120
+export const ANIMATION_DURATION = 1000
+
+const Container: StyledComponent<{ negativeAction: boolean }, ThemeType, *> = styled(Animated.View)`
```

Comment:
as far as I can see `negativeAction` is always true

```suggestion
const Container: StyledComponent<{ negativeAction: boolean }, ThemeType, *> = styled(Animated.View)`
```

Reply:
Ahh yeah, totally forgot that. I tried to make it more customizable at first by making the negativeAction optional.